draft-ietf-v6ops-prefer8781.md

```diff
@@ -46,103 +46,108 @@ informative:
-  * translate the IPv4 literal to an IPv6 literal (Section 7.1 of [RFC8305]);
-  * perform local DNS64 ([RFC6147]) functions.
+  * Implement the customer-side translator (CLAT) function of the 464XLAT architecture [RFC6877];
+  * Translate IPv4 literals to IPv6 literals (Section 7.1 of [RFC8305]);
+  * Perform local DNS64 [RFC6147] functions.
 
-Dynamic PREF64 discovery is often essential, particularly for unmanaged or mobile endpoints, where static configuration is impractical.
-While [RFC7050] introduced the first DNS64-based mechanism for PREF64 discovery, subsequent methods have been developed to address its limitations.
+Dynamic PREF64 discovery is useful to keep the NAT64 prefix configuration up-to-date, particularly for unmanaged endpoints or endpoints which move between networks.
+[RFC7050] introduces the first DNS64-based mechanism for PREF64 discovery based on [RFC7051] analysis.
+However, subsequent methods have been developed to address [RFC7050] limitations.
 
 
-For instance, [RFC8781] defines a Neighbor Discovery ([RFC4861]) option for Router Advertisements (RAs) to convey PREF64 information to hosts.
+For instance, [RFC8781] defines a Neighbor Discovery [RFC4861] option for Router Advertisements (RAs) to convey PREF64 information to hosts.
 This approach offers several advantages (Section 3 of [RFC8781]), including fate sharing  with other host network configuration parameters.
 
 Due to fundamental shortcomings of the [RFC7050] mechanism ({{issues}}), [RFC8781] is the preferred solution for new deployments.
 Implementations should strive for consistent PREF64 acquisition methods.
-The DNS64-based mechanism of [RFC7050] should be employed only when RA-based PREF64 delivery is unavailable, or as a fallback for legacy systems incapable of processing the PREF64 RA option.
+The DNS64-based mechanism of [RFC7050] should be employed only when RA-based PREF64 delivery is unavailable, or as a fallback for legacy systems incapable of processing the PREF64 RA Option.
 
 # Conventions and Definitions
 
-CLAT: A customer-side translator (XLAT), defined in [RFC6877], that complies with [RFC7915].
-
 DNS64: a mechanism for synthesizing AAAA records from A records, defined in [RFC6147].
 
 NAT64: a mechanism for translating IPv6 packets to IPv4 packets and vice versa.  The translation is done by translating the packet headers according to the IP/ICMP Translation Algorithm defined in [RFC7915]. NAT64 translators can operate in stateless or stateful mode ([RFC6144]).
 
-PREF64 (or NAT64 prefix): An IPv6 prefix used for IPv6 address synthesis and for network addresses and protocols translation from IPv6 clients to IPv4 servers, [RFC6146].
+PREF64 (or Pref64::/n, or NAT64 prefix): An IPv6 prefix used for IPv6 address synthesis and for network addresses and protocols translation from IPv6 clients to IPv4 servers using the algorithm defined in [RFC6052].
```

Comment:
Same addition of the RFC description here added in the abstract for 7050

Reply:
The problem of RFC6052 is that its title is 'IPv6 Addressing of IPv4/IPv6 Translators' and it does define the translation algorith - so I'm struggling with adding a meaningful description w/o making the text awkward.